packages/nitro-protocol/src/contract/state.ts

```diff
@@ -4,7 +4,7 @@ import {encodeOutcome, hashOutcome, Outcome} from './outcome';
 import {Address, Bytes32, Uint256, Uint48} from './types';
 
 export interface State {
-  turnNum: number;
+  turnNum: number; // TODO: This should maybe be a string b/c it is uint256 in solidity
```

Comment:
While turnNum is currently a `uint256` it sounds like we may be limiting it to `uint48` per https://research.statechannels.org/t/how-large-does-the-largest-turnnum-need-to-be/35, which would allow us to keep `turnNum` as a `number`.  I think it makes sense to keep the TODO for now and address this in the future.